README.md

```diff
@@ -204,6 +204,20 @@ my-s3-bucket:
         HEALTHCHECK WORKS!
 ```
 
+## Postgres database
```

Comment:
is it just worth adding a line similar to the ones below opensearch / s3 to say that this resource is created via the `extensions.yml` file etc.?